### ISSUE TITLE
[Bug]: batch playback errors when there are no files with an accurate anime name

### ISSUE THREAD

**Issue by nlxdodge** [ThaUnknown/miru#203]
### Preflight checklist

- [X] I have searched the [issue tracker](https://github.com/ThaUnknown/miru/issues) for a bug report that matches the one I want to file, without success.
- [X] I have searched the [frequently asked questions](https://github.com/ThaUnknown/miru/blob/master/docs/faq.md) for a solution that fixes this problem, without success.
- [X] I have checked that I'm using the [latest stable](https://github.com/ThaUnknown/miru/releases/latest) version of the app.

### What app version are you using?

v2.13.3

### What operating system version are you using?

Windows

### Expected Behavior

When (in my example) I try to play [This](https://anilist.co/anime/107693/Mairimashita-Irumakun/) anime. It did a batch lookup in another version of Miru and that worked great.

So I wanted to continue with episode 8 so I click on continue watching and the title also shows Episode 8.

### Actual Behavior

However it plays a different episode, when clicking next or previous or any other episode from the list. It just starts the same episode over again.
When I looked at the name I found out that it always plays episode 21 (of the 23 in the season).

Is there a cache somewhere that I can delete to maybe try to fix this? I already tried to download an older version, but it auto upgrades to the latest version.
And I wanted to turn of batching again, but it seems an update has removed the button.

### Screenshots

![image](https://user-images.githubusercontent.com/8994983/182467828-d63502c2-74dc-4b41-9c04-a7220797a73d.png)

**Comment by ThaUnknown** [comment]
does it actually constantly play EP 8, or does it actually play a different EP while saying it plays EP 8?

**Comment by ThaUnknown** [comment]
I see the issue, and I'm not certain how to solve it
![image](https://user-images.githubusercontent.com/6506529/182474147-b1b729ca-241d-4df4-b775-9c2bd50f3546.png)

TLDR some idiot decided to not include the anime name in the files, so this needs to be adressed somehow, I assume that's what causes the error

**Comment by ThaUnknown** [comment]
@nlxdodge you can work around this by turning off `autoplay torrents` in the settings, then picking a different torrent, batch support option is removed as it by itself is [close to] flawless now, and it doesn't exclude single episode playback from the torrent list see:
![image](https://user-images.githubusercontent.com/6506529/182479990-6b125325-d51d-4012-9568-249e49722831.png)
